VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html

```diff
@@ -237,7 +237,7 @@ <h2 class="govuk-heading-m" *ngIf="isNotStarted()">Is everyone connected?</h2>
     </div>
 </div>
 
-<app-participant-chat *ngIf="isIMEnabled && !isAdminConsultation" [hearing]="hearing" [alwaysOn]="hearing.isInSession()"></app-participant-chat>
+<app-participant-chat *ngIf="defineIsIMEnabled() && !isAdminConsultation" [hearing]="hearing" [alwaysOn]="hearing.isInSession()"></app-participant-chat>
```

Comment:
why does this work and using `isIMEnabled` doesnt?

do we still need `isIMEnabled`? it has no other uses, and looks like it doesnt work? if not could we remove, please :)

Reply:
Removed redundant code